scripts/createSecrets.js

```diff
@@ -0,0 +1,12 @@
+const crypto = require('crypto');
```

Comment:
I guess this is meant as a utility that someone could run from the command line? Does it make sense to have it as a function that returns the two values so that it can be used in other scripts if desired (like a startup script).

Reply:
This is meant to be a script ran from the command line since its in the `scripts` directory

Reply:
it could log and return though right? That way someone could use the values in a startup script or something if they wanted to.

Reply:
Updated to make this file requireable